lib/src/main/kotlin/uk/gov/justice/hmpps/sqs/HmppsQueue.kt

```diff
@@ -3,6 +3,7 @@ package uk.gov.justice.hmpps.sqs
 import com.amazonaws.services.sqs.AmazonSQS
 
 class HmppsQueue(
+  val id: String,
```

Comment:
Added an identifier for each queue appearing in queue health.  Will be taken from the queue config map keys in a future PR.